internal/configs/version_constraint.go

```diff
@@ -31,6 +31,8 @@ func decodeVersionConstraint(attr *hcl.Attribute) (VersionConstraint, hcl.Diagno
 	}
 	return decodeVersionConstraintValue(attr, val)
 }
+
+// TODO ephemeral - check how the ephemeral mark should be handled in this function
```

Comment:
Should be forbidden, same as sensitive

Reply:
Fixed in 7e8efcacaf5b7cbcfbe33eed4b5f5593331b4519